### ISSUE TITLE
Slider (option) in Climate card

### ISSUE THREAD

**Issue by DohmenICT** [piitaya/lovelace-mushroom#805]
### Is your feature request related to a problem?

When I have a thermostat with a .1 degree granularity, I need to click at least 10 times on the + button to increase the temp setting with 1 degree. On mobile devices a double-click will zoom in the UI. A slider would fix both problems.

### Describe the solution you'd like

Changing the temp setting on a climate card could also be done with a slider, just as the dimmer cards do.

Would be nice to have a custom mix-max and granularity setting in the UI to override the standard Climate settings of the device we’re configuring. 

### Describe alternatives you've considered

I’m using the default HA climate card now. But I’d rather use the Mushroom card.

**Comment by Strauman** [comment]
I totally agree @DohmenICT! How awesome and slick it would be to just simply slide the temp instead of clicking. Maybe even vertically too? Then you could put It side by side and slide temp up and down like a pro. It'd be so awesome if someone implemented this.

**Comment by aslabicki** [comment]
+1

To me, that slider option would make the card more awesome

**Comment by tonivss** [comment]
I am also very interested in this feature!

**Comment by agak79** [comment]
+1 I think this would be a fancy feature, very useful for a mobile dashboard. There is enough space in the card to include the slide
